examples/ssx-test-nextauth-email/components/SignInWithEmailModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+interface ISignInWithEmailModal {
+  title?: string,
+  showModal: boolean,
+  handleClose: () => void,
+  handleEmailSignIn: (email: string) => void;
+}
+
+const SignInWithEmailModal = ({ title = 'Sign-In with Email', showModal, handleClose, handleEmailSignIn }: ISignInWithEmailModal) => {
```

Comment:
Is there a way in the modal to add a small text once clicked as `Check your email for sign in instructions` rather than closing it out